src/shared/Table/TableView.tsx

```diff
@@ -96,6 +104,39 @@ export const TableView = <TRow, TCol>({
   if (data?.length === 0 && !isFiltered) {
     return emptyStateNoData;
   }
+
+  const transformDropdownItems = () => {
+    const buttonActions = actions?.map((a, id) => (
+      <OverflowMenuDropdownItem
+        key={id}
+        onClick={() => {
+          a.onClick;
+          toggleIsActionsOpen(false);
+        }}
+        isShared
+      >
+        {a.label}
+      </OverflowMenuDropdownItem>
+    ));
+    const kebabActionItems = kebabActions?.map((a, id) => (
+      <OverflowMenuDropdownItem
+        key={id}
+        onClick={() => {
+          a.onClick;
+          toggleIsActionsOpen(false);
+        }}
+        isDisabled={a.isDisabled}
+      >
+        {a.label}
+      </OverflowMenuDropdownItem>
+    ));
+    if (buttonActions != undefined && kebabActionItems != undefined)
+      return buttonActions.concat(kebabActionItems);
+    else if (buttonActions != undefined) return buttonActions;
+    return kebabActionItems;
```

Comment:
```suggestion
    if (buttonActions != undefined && kebabActionItems != undefined)
      return buttonActions.concat(kebabActionItems);
    else if (buttonActions != undefined) return buttonActions;
    return [
       ...(buttonActions || []),
       ...(kebabActionItems || []),
    ];
```

`concat` mutates the original array in place. I'd suggest avoiding using it unless there is a good reason for not doing so.